### PR TITLE
Added union type definition support

### PR DESCRIPTION
**Pull request by walangkaji**
[#5](https://github.com/rethinkphp/json-validator/pull/5#issue-313066891)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,11 +48,12 @@ $validator->defineType('User', [
     'name' => 'string',
     'gender' => 'string',
     'age' => '?integer',
+    'rating' => '?integer,?boolean',
```
Comment (bixuehujin) [inline README.md:51]: The syntax should be like this: `?integer|boolean`, the question mark indicates the field is nullable, and `|` indicates the field accepts an integer or boolean.

**Comment by bixuehujin** [comment]
BTW, Can you add a test case for this?

File: README.md (hunk 1 above)
Comment (walangkaji) [reply README.md:51]: nice, let me try

**Comment by walangkaji** [comment]
hi @bixuehujin 
check is there any mistake

**Comment by bixuehujin** [comment]
lgtm, merged

**Pull request merged**